README.md

```diff
@@ -47,3 +47,37 @@ myServer.run({
   })
 })
 ```
+
+## Docs :
+
+### `const server = new SQSServer()`
+
+Create a fake SQS server
+
+### `server.run(opts, cb)`
+
+ - `opts.port` ; defaults to 0
+ - `opts.host` ; defaults to `localhost`
+
+Starts the server. cb get's called once listening on a port.
+
+### `server.getQueue()`
+
+Returns the current array of items queued in SQS
+
+### `server.waitForMessages(count, listener)`
+
+Get notified once N messages have been send to this fake SQS
```

Comment:
`listener` is a callback, and not an eventemitter listener that can be called multiple times, right?

Reply:
it's not a `cb` because there's never an error. It's a `listener` that gets called once.